### ISSUE TITLE
Swagger 1.2 Documentation missing info section

### ISSUE THREAD

**Issue by sankarbalu** [springfox/springfox#1197]
I am using springfox library version 2.3.0 along with spring-boot-1.2.5.RELEASE. I am able to generate Swagger 2 schema without any issues. However there is a requirement to generate the schema in Swagger 1.2 format as well. After reading the documentation and several posts, I managed to get the Swagger2 and Swagger12 schema co-exist using separate Docket definitions. I am able to see both swagger 1.2 and swagger 2 versions of the schema in the swagger-ui as well. 

However, in the Swagger 1.2 version of the schema, i see that the info section is missing, which i would expect to be as below

```json
 "info": {
  "title": "test title",
  "description": "sample description",
  "contact": "abcd@test.org"
  } 
```

Also, the produces section is present, but is blank as below. I would expect it to have 
```json
"application/json".
 "produces": [
   
  ]
```

My Swagger configuration is as below

```java
@Configuration
@EnableSwagger
@EnableSwagger2
public class SwaggerConfig {

	@Bean
	public Docket swagger2Api() {
		return new Docket(DocumentationType.SWAGGER_2)
				.groupName("group-1")
				.apiInfo(sampleApiInfo())
				.select()
                                .apis(RequestHandlerSelectors.basePackage("com.xyz.controller"))
				.build();
	}

	@Bean
	public Docket swagger12Api() {
		return new Docket(DocumentationType.SWAGGER_12)
				.apiInfo(sampleApiInfo())
				.produces(new HashSet<String>(Arrays.asList("application/json")))
				.forCodeGeneration(true)
				.select()
                                .apis(RequestHandlerSelectors.basePackage("com.xyz.controller"))
				.build();
	}

	private ApiInfo sampleApiInfo() {
		return new ApiInfoBuilder().title("sample title")
				.description("sample description")
				.termsOfServiceUrl("sample-url).contact("abcd").license("xyz)
				.licenseUrl("https://xyz.com/LICENSE").version("0.1").build();
	}
	
```
Am i missing something here?

**Comment by dilipkrish** [comment]
@sankarbalu Not really sure. It should work just fine and there are [functional tests](https://github.com/springfox/springfox/blob/master/swagger-contract-tests/src/test/groovy/springfox/test/contract/swaggertests/FunctionContractSpec.groovy) that do exactly what you're doing

**Comment by sankarbalu** [comment]
@dilipkrish The functional test do not seem to check the info section or the produces element in the result. May be if these tests are included, i guess they might fail.

**Comment by dilipkrish** [comment]
It [totally](https://github.com/springfox/springfox/tree/master/swagger-contract-tests/src/test/resources/contract/swagger) [does](https://github.com/springfox/springfox/blob/master/swagger-contract-tests/src/test/resources/contract/swagger/resource-listing.json#L115-L122). Not sure what you're talking about

**Comment by sankarbalu** [comment]
My bad. I didn't read the Swagger 1.2 specification completely. 

After RTFM (https://github.com/OAI/OpenAPI-Specification/blob/master/versions/1.2.md), I realized that the info section is supposed to be present only under the resource listing endpoint at /api-docs, and not at the /api-docs/default/<resource> where i was looking for. 

Also the other issue with the empty produces section was resolved when I added the produces and consumes builder methods while generating the ApiInfo. 

Thanks a lot for your time @dilipkrish. Appreciate it.
